notebooks/Bruxism_and_MEMA_combine.ipynb

```diff
@@ -10,18 +10,14 @@
   },
```

Comment:
<p>I remember now that we should also check why there were cases with combine bruxism without associated combine mema episode. it shouldn't exist!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lkorczowski/Tinnitus-n-Sleep/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>